GameEngine/GameState.cs

```diff
@@ -1,14 +1,19 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace GameEngine
 {
     public class GameState
     {
         public GameBoard Board;
         public IDeck Deck;
-        public PlayerHand Hand;
+        public List<PlayerHand> Hands;
```

Comment:
Is `IList` the convention for C#?

Reply:
IList has an unfortunate interface ... although I don't think we're using anything from List that isn't already on IList.  So I don't think making the change is a problem.

Reply:
👍